estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class PropertyOffer(models.Model):
```

Comment:
```suggestion
class EstatePropertyOffer(models.Model):
```
The class name should match the name of the model